BigQueryDataTransfer/src/V1/DataTransferServiceClient.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
- * Copyright 2017 Google LLC
+ * Copyright 2018 Google LLC
```

Comment:
Just a friendly reminder we don't need to update these

Reply:
Hmm, yes, need to fix tooling around this. Will update.